### PR TITLE
Feature/LP-2: Library (Repository): add delete functionality

### PR DESCRIPTION
**Pull request by nlitvins**
deleteById and tests ready

### CODE REVIEW

File: src/main/java/Library.java [hunk 1]
```diff
@@ -10,6 +10,9 @@ public Library() {
         books.add(new Book(1232, "Shrek", "John", true));
         books.add(new Book(1233, "Lord Farquaad", "Jack", true));
         books.add(new Book(1234, "Prince Charming", "Ivan", false));
+
+        books.remove(1);
```
Comment (dlitvins) [inline src/main/java/Library.java:14]: why?

File: src/main/java/Library.java [hunk 2]
```diff
@@ -22,11 +25,22 @@ public Book findById(int searchId) {
             if (searchId == book.getId()) {
                 return book;
             }
+
```
Comment (dlitvins) [inline src/main/java/Library.java:28]: why?

**Review by dlitvins — CHANGES_REQUESTED** [review]

File: src/main/java/Library.java (hunk 1 above)
Comment (nlitvins) [reply src/main/java/Library.java:14]: done

File: src/main/java/Library.java (hunk 2 above)
Comment (nlitvins) [reply src/main/java/Library.java:28]: done

**Pull request merged**